src/plugins/datasource.ts

```diff
@@ -63,8 +66,9 @@ export const AppDataSource = new DataSource({
   poolSize: DB_CONNECTION_POOL_SIZE, // *2 because of the number of tasks
   // log queries that take more than 2s to execute
   maxQueryExecutionTime: 2000,
+  // needs to change based on where it runs
   logging: ['migration', 'error'],
-  migrationsRun: true,
+  // migrationsRun: true,
```

Comment:
WARNING: Currently migrations will not be run automatically on start.

Reply:
As long as we don't have a solution for migration, I would let it be, so in case of easy migrations it can run smoothly.